src/util/std_types.h

```diff
@@ -1051,8 +1051,8 @@ class incomplete_array_typet:public type_with_subtypet
   {
   }
 
-  explicit incomplete_array_typet(const typet &_subtype):
-    type_with_subtypet(ID_array, _subtype)
+  explicit incomplete_array_typet(const typet &_subtype)
```

Comment:
So this is a change in behaviour - have you checked that this constructor wasn't used anywhere? CLion says no, so you might just like to delete it?

Reply:
It seems like it's not used indeed, removed.